packages/components/buttons/icon-button/src/icon-button.tsx

```diff
@@ -59,7 +59,7 @@ export type TIconButtonProps<
     event: MouseEvent<HTMLButtonElement> | KeyboardEvent<HTMLButtonElement>
   ) => void;
   /**
-   * The container shape of the button.
```

Comment:
I removed this prop description as I noticed that none of the deprecated prop have one.